_docs/floodgate/linking.md

```diff
@@ -16,6 +19,8 @@ Global Linking is part of the [Global Api](https://wiki.geysermc.org/floodgate/f
 4. Enter the random number on the other account by typing `/linkaccount <code>`
 5. You should get kicked from the server on both your Bedrock and Java account with the message that it had successfully linked your accounts.
 
+To unlink your accounts, join the GlobalLinkServer (as described above for linking) on either Java or Bedrock, and use the /unlinkaccount command. 
```

Comment:
and use the -> or use the

Reply:
But i'm trying to describe how globally unlink? I've added "To unlink your globally linked accounts[..]" at the beginning
Cuz you cannot unlink globally linked accounts on the website, and the "unlinkaccount" for global linking only exists on the GlobalLinkServer

Reply:
Oh, I see now. I was thinking about the local `/unlinkaccount` command on Floodgate servers.